velox/dwio/dwrf/common/Compression.cpp

```diff
@@ -433,38 +434,38 @@ bool ZlibDecompressionStream::Next(const void** data, int32_t* size) {
 } // namespace
 
 std::unique_ptr<BufferedOutputStream> createCompressor(
-    CompressionKind kind,
+    dwio::common::CompressionKind kind,
```

Comment:
@frankobe Is there any reason why the CompressionKind was moved, but the compressor and decompressor were not? Are you going to move them in later PRs?